Content.Shared/Clumsy/ClumsyComponent.cs

```diff
@@ -58,4 +58,46 @@ public sealed partial class ClumsyComponent : Component
     /// </summary>
     [DataField]
     public SoundSpecifier GunShootFailSound = new SoundPathSpecifier("/Audio/Weapons/Guns/Gunshots/bang.ogg");
+	
+	/// <summary>
+	///		Imp. Whether or not to apply Clumsy to hyposprays.
```

Comment:
What does the "Imp." mean?

Reply:
The name of our private server. Had that in there to mark changes that are exclusive to our server, so that in the event of a future merge conflict it's easier to see what changes are ours. I can remove those from the upstream PR if you want.

Reply:
I removed it, just in case.